### ISSUE TITLE
Adding services inside app class

### ISSUE THREAD

**Issue by ste7een** [EnabApp/core#2]
We need to adding services for each app inside app class with "owned or not" , in **appManager**.  

```
app=[
.
.
 services:[
      id,
      title,
      points,
      description,
      icon,
      owned,
 ]

]
```

**Comment by Gruce** [comment]
Hello @ste7een,

Done! please look at the wiki of `core` for `composables` section and `models`. Reopen this issue if any new update needed!

Thank you,

**Comment by ste7een** [comment]
Hi @Gruce 
I cant find the description field in service and app class , please check it in you classes or `composables ` .
Thank you .

**Comment by Gruce** [comment]
Hello @ste7een,

Sorry for the inconvenience.

Solved at `enab@core@v1.8.5-beta.20`

Thankyou
